lib/kinetic_cafe/error_dsl.rb

```diff
@@ -1,7 +1,7 @@
-# frozen_string_literal: false
+# frozen_string_literal: true
 module KineticCafe
```

Comment:
Please change this to:

```ruby
# frozen_string_literal: true

##
module KineticCafe
```

It will get around a particularly annoying rdoc bug that doesn’t ignore `frozen_string_literal` in some versions.